.github/workflows/standard-workflow.yml

```diff
@@ -0,0 +1,161 @@
+name: Standard Workflow
+
+on: workflow_call
+
+jobs:
+  code-style-check:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out
+        uses: actions/checkout@v2
+
+      - name: Setup Scala
+        uses: olafurpg/setup-scala@v10
+        with:
+          java-version: adopt@1.11
+
+      - name: Resolve dependencies
+        run: sbt update
+
+      - name: Run code checks
+        run: |
+          PATH=$PATH:./scripts
+          set -x
+          enforce-newline-at-eof
+          prohibit-trailing-whitespace
+          enforce-issue-number-for-todos
+
+          npm install --global remark-cli remark-lint-no-dead-urls remark-validate-links
+          validate-markdown-links
+
+      - name: Check code style
+        # Curiously, scalafmtCheckAll doesn't cover *.sbt files, hence scalafmtSbtCheck is needed as well
+        run: sbt scalafmtCheckAll scalafmtSbtCheck "scalafixAll --check"
+
+  test-212:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out
+        uses: actions/checkout@v2
+
+      - name: Setup Scala
+        uses: olafurpg/setup-scala@v10
+        with:
+          java-version: adopt@1.11
+
+      - name: Resolve dependencies
+        run: sbt update
+
+      - name: Compile
+        run: sbt compile Test/compile
+
+      - name: Run tests
+        run: |
+          sbt "++ 2.12.13 test"
+          sbt "++ 2.12.14 test"
+          sbt "++ 2.12.15 test"
+
+  test-213-1:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out
+        uses: actions/checkout@v2
+
+      - name: Setup Scala
+        uses: olafurpg/setup-scala@v10
+        with:
+          java-version: adopt@1.11
+
+      - name: Resolve dependencies
+        run: sbt update
+
+      - name: Compile
+        run: sbt compile Test/compile
+
+      - name: Run tests
+        run: |
+          sbt "++ 2.13.2 test"
+          sbt "++ 2.13.3 test"
+          sbt "++ 2.13.4 test"
+          sbt "++ 2.13.5 test"
+
+  test-213-2:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out
+        uses: actions/checkout@v2
+
+      - name: Setup Scala
+        uses: olafurpg/setup-scala@v10
+        with:
+          java-version: adopt@1.11
```

Comment:
Nit: is there some neat way of extracting these repeatable 2 steps (like e.g. in CircleCI: https://github.com/VirtusLab/git-machete-intellij-plugin/blob/master/.circleci/config.yml#L2)? if no, let's leave as is

Reply:
Looks like it might be a problem: https://github.community/t/support-for-yaml-anchors/16128/88 - so let me leave it as it is for now